bridge/src/lib.rs

```diff
@@ -849,10 +910,29 @@ pub mod pallet {
 
 		/// unpause all registered domains in the storage
 		fn unpause_all_domains() {
+			// iterate over the registered domain storage first and write all of them into IsPaused
+			// storage
+			for registered_domain_pair in DestDomainIds::<T>::iter() {
+				IsPaused::<T>::insert(registered_domain_pair.0, false);
+			}
```

Comment:
```suggestion
			IsPaused::<T>::translate(|_, _| Some(false));
```
This way should be more effective

Reply:
I tried with `translate` method but it requires adding `#![recursion_limit = "256"]` attribute to bridge crate, so I did, and it keeps complaining about recursion limit all the way to 131072, then the compile time takes more than 2 hours, so I shut it off. the Error is shown in the below screenshot:
<img width="588" alt="image" src="https://github.com/sygmaprotocol/sygma-substrate-pallets/assets/13714909/45c54841-c0d6-4260-a4de-606462e06d42">

I guess it has something to do with the Decode trait that translate is using?

Reply:
@tolak I pushed the fix for this comment, please have another look 🙏

Reply:
Nice job!